tests/raw_query/models.py

```diff
@@ -23,6 +23,10 @@ class Book(models.Model):
     opening_line = models.TextField()
 
 
+class BookInfo(models.Model):
```

Comment:
I would call it somethign like `BookFkAsPk` so it's more descriptive about the purpose of the test model.